website/docs/r/ad_secret_backend.html.md

```diff
@@ -86,6 +86,7 @@ Defaults to `false`.
 shows a later rotation, it should be considered out-of-band
 
 * `length` - (Optional) **Deprecated** use `password_policy`. The desired length of passwords that Vault generates.
+  *Mutually exclusive with `password_policy` on vault-1.11+*
```

Comment:
Should we add a line on mutual exclusivity for the formatter field as well?

Reply:
I think it is okay to leave it as-is since the provider is taking care of enforcing the policy on `formatter`.